README.md

```diff
@@ -18,7 +17,19 @@ To get the source code of our SDKs and samples via **git** just type:
     git clone https://github.com/Azure/azure-mobile-services.git
     cd ./azure-mobile-services/
 
-## Change log
+## SDk Documentation references
+
+#### Azure App Service Mobile Apps
+* [iOS](http://azure.github.io/azure-mobile-services/ios/v3)
+* [Managed](http://azure.github.io/azure-mobile-services/managed/v2)
+
+#### Mobile Services
+* [iOS](http://azure.github.io/azure-mobile-services/ios/v2)
+* [Managed](http://azure.github.io/azure-mobile-services/managed/v1)
+* [Android](http://azure.github.io/azure-mobile-services/android/v2)
+* [Javascript](http://azure.github.io/azure-mobile-services/javascript/v1)
+
+## SDK Change log
```

Comment:
It might be less confusing for us to differentiate between vnext and mobile services. What do you guys think? Simply giving the versions might be confusing for customers who don't know of App Service @lindydonna